dev-python/icalendar/icalendar-4.0.6.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
3.9? ;-)  If you don't have it, I'll just test it myself.

Reply:
`dev-python/python-dateutil` does not have 3.9.

Reply:
Sorry, what I meant is when I initially submitted this PR it didn’t. It does now so I’ll test it.

Reply:
Rebased and added 3.9 in 107225d4ac7c. The unit tests pass, so hopefully that means it works.